### PR TITLE
socket.io connection established

### PR DESCRIPTION
**Pull request by ashjd**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+    });
+  </script>
 
     <!-- initial react app tag -->
     <div id="app"></div>
 
     <!-- this is where the webpack.config serves the files, see output publicPath -->
     <script type="text/javascript" src="bundle.js"></script>
+   
+   <!--<script src="/socket.io/socket.io.js"></script>-->
```
Comment (david-doan) [inline client/index.html]: remove comment

File: client/components/App.js [hunk 2]
```diff
@@ -2,6 +2,12 @@ import React, { PropTypes, Component } from 'react';
 
 
 class App extends Component {
+
+  componentDidMount() {
+    console.log('mounted');
+    var socket = io();
```
Comment (david-doan) [inline client/components/App.js]: might be redundant with like  9 of client/index.html

File: server/index.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (david-doan) [inline server/index.html]: what is this file for? is it required?

File: client/bundle.js [hunk 4]
```diff
@@ -19708,6 +19708,12 @@
 	  }
 
 	  _createClass(App, [{
+	    key: 'componentDidMount',
```
Comment (david-doan) [inline client/bundle.js:19711]: i thought we .gitignored this file. We may have to either remove it or have git stop tracking it.
to remove: git rm --cached client/bundle.js
to keep, but stop tracking changes: git update-index --assume-unchanged client/bundle.js

Comment (ashjd) [inline client/bundle.js:19711]: I removed the cached version

**Pull request merged**